### DISCUSSION TITLE
"My Fork" button

### DISCUSSION THREAD

**Discussion by ds5678** [community/community#18990] [Other Feature Feedback, Questions, & Ideas] [answered]
It would be convenient to have an easy way to navigate to my personal/organization fork. Currently, for repositories with a large number of forks, it is cumbersome to navigate from the origin repository to my forked repository.

**Comment by doguedogue** [comment]
A button would be usefull but I think now you can navigate with the links that shows from where is forked and in the original repo you can navigate to your repo showing all the forks. (web page)

**Reply by ds5678** [reply]
> showing all the forks

For repositories with a large number of forks, only a subset is shown.

**Comment by aisgbnok** [comment, marked as answer]
[The repository fork button now displays existing forks.](https://github.blog/changelog/2022-06-29-the-repository-fork-button-now-displays-existing-forks/) 🎉

**Reply by verhovsky** [reply]
Didn't it used to be that clicking "Fork" when you already had a personal fork would just take you there? We went from 1 click to 1st click + wait ~700ms + choose from a list with 1 item in it + 2nd click for like one of the most common actions I need to take when contributing to a project.

**Reply by aisgbnok** [reply]
I don't think so. I remember it providing just a massive list of all the forks, so I stopped using the button. Although that might have been a while ago. You might be right.

A refinement of the current experience could be: that clicking the fork button takes you to your fork if you already have one. If you have more than one, you can select a default in that list.
